docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,44 @@
+## Overview
+
+AI DIAL Analytics Realtime is a service that gathers prompt logs from [DIAL Core](https://github.com/epam/ai-dial-core) and publishes them to InfluxDB.
+The logs of prompts are the data a user provides to LLM and the responses the LLM gives to the user. A user can access analytical data through pre-set Grafana dashboards.
+
+**TODO**
```

Comment:
What is the status of this?

Reply:
The project needs to be updated after completion of the PR.
We will just include a link to the doc in analytics realtime README